examples/call-service-panel-example/package.json

```diff
@@ -39,5 +39,11 @@
     "react-dom": "18.2.0",
     "react-json-view": "^1.21.3",
     "typescript": "4.9.5"
+  },
+  "overrides": {
+    "react-json-view": {
+      "react": "$react",
+      "react-dom": "$react-dom"
+    }
```

Comment:
What's going on here?

Reply:
Unfortunately react-json-view requires react 17. Yarn seems to resolve this without any extra help but I had to add this to make npm happy.

Reply:
Seems like if react-json-view doesn't support react 18 then we shouldn't use 18 in the template, or we should find a different library to use that does support 18? I wouldn't expect anything to work properly if we are using 18 when the library's not compatible...

Reply:
The last time I tried this panel it seemed to still work. There is also a react-json-view fork that does support 18 but didn't seem to have a lot of traction. I could try that.

Reply:
Decided to just use react 17 for the examples that depend on react-json-view. We can find a replacement in a followup.